app/models/rent.rb

```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
+  validates :user_id, :book_id, :from, :to, presence: true
```

Comment:
There is no need to validate the presence of `:user_id` and `:book_id` because since Rails 5 every `belongs_to` relation is mandatory